dracut/30ignition/coreos-teardown-initramfs-network.sh

```diff
@@ -20,7 +20,10 @@ schedule_selinux_relabel() {
 #
 # See https://github.com/coreos/fedora-coreos-tracker/issues/394#issuecomment-599721173
 propagate_initramfs_networking() {
-    if [ -n "$(ls -A /sysroot/etc/NetworkManager/system-connections/)" ]; then
+    # Check the two locations where a user could have provided network configuration
+    # On FCOS we only support keyfiles, but on RHCOS we support keyfiles and ifcfg
+    if [ -n "$(ls -A /sysroot/etc/NetworkManager/system-connections/)" -o \
+         -n "$(ls -A /sysroot/etc/sysconfig/network-scripts/)" ]; then
```

Comment:
Shouldn't this be conditional on being on RHCOS?

Reply:
yes... but I'm lazy 🦥. I just figured since ifcfg files don't work on FCOS no one is going to be using them and we can be ok without the extra checking.

Reply:
Sure, WFM!